js/bookmark.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty files should be removed from your PR as they are not needed for the functionality of your app.